src/components/diagrams/networkAreaDiagram/network-area-diagram-content.tsx

```diff
@@ -144,12 +145,16 @@ function NetworkAreaDiagramContent(props: NetworkAreaDiagramContentProps) {
     const OnLeftClickCallback: OnSelectNodeCallbackType = useCallback(
         (equipmentId, nodeId, mousePosition) => {
             if (mousePosition && !props.loadingState) {
-                setSelectedVoltageLevelId(equipmentId);
-                setShouldDisplayMenu(true);
-                setMenuAnchorPosition(mousePosition ? { mouseX: mousePosition.x, mouseY: mousePosition.y } : null);
+                if (isEditNadMode) {
+                    setSelectedVoltageLevelId(equipmentId);
+                    setShouldDisplayMenu(true);
+                    setMenuAnchorPosition(mousePosition ? { mouseX: mousePosition.x, mouseY: mousePosition.y } : null);
+                } else {
+                    dispatch(openDiagram(equipmentId, DiagramType.VOLTAGE_LEVEL));
```

Comment:
Not sure it's a good idea to pass throught the redux store here.
We already pass callback props here to manage actions: see ` onExpandVoltageLevel` and `onHideVoltageLevel`.
We should just add a `onVoltageLevelClick` prop callback I think